journalist_gui/journalist_gui/resources_rc.py

```diff
@@ -1652,10 +1652,15 @@
     rcc_version = 2
     qt_resource_struct = qt_resource_struct_v2
 
+
 def qInitResources():
-    QtCore.qRegisterResourceData(rcc_version, qt_resource_struct, qt_resource_name, qt_resource_data)
+    QtCore.qRegisterResourceData(rcc_version, qt_resource_struct,
```

Comment:
The resources file is generally autogenarated,  I think we should add this into the `flake8` exclude list.

Reply:
Ah good call, fixing...

Reply:
OK fixed and repushed. Feel free to formally approve this one if it looks good (since it's into our shared branch `qt-journalist-updater`)